_config.yml

```diff
@@ -151,6 +151,7 @@ alias:
 
   # obsolete content
   dashboard/overview/runs-dashboard.html: guides/dashboard/runs.html
+  guides/core-concepts/launching-browsers.html: guides/guides/launching-browsers.html#Browsers
```

Comment:
These should not include hashes - they should directly point to an existing document in this repo.

```suggestion
  guides/core-concepts/launching-browsers.html: guides/guides/launching-browsers.html
```

Reply:
Thanks, done.